projectDocs/dev/codingStandards.md

```diff
@@ -3,17 +3,20 @@
 Python code style is enforced with the flake8 linter, see [`tests/lint/readme.md`](https://github.com/nvaccess/nvda/tree/master/tests/lint/readme.md) for more information.
 
 ### Encoding
+
 * Where Python files contain non-ASCII characters, they should be encoded in UTF-8.
-    * There should be no Unicode BOM at the start of the file, as this unfortunately breaks one of the translation tools we use (`xgettext`).
-    Instead, include this as the first line of the file, only if the file contains non-ASCII characters:
-        ```py
-        # -*- coding: UTF-8 -*-
-        ```
-    * This coding comment must also be included if strings in the code (even strings that aren't translatable) contain escape sequences that produce non-ASCII characters; e.g. `"\xff"`.
+  * There should be no Unicode BOM at the start of the file, as this unfortunately breaks one of the translation tools we use (`xgettext`).
+  Instead, include this as the first line of the file, only if the file contains non-ASCII characters:
+
+  ```py
+  # -*- coding: UTF-8 -*-
+  ```
+
+  * This coding comment must also be included if strings in the code (even strings that aren't translatable) contain escape sequences that produce non-ASCII characters; e.g. `"\xff"`.
     This is particularly relevant for braille display drivers.
     This is due to a `gettext` bug; see [comment on #2592](https://github.com/nvaccess/nvda/issues/2592#issuecomment-155299911).
-* New files should contain `LF` line endings, however NVDA currently uses a mix of LF and CRLF line endings.
-See issue [#12387](https://github.com/nvaccess/nvda/issues/12387).
+* Most files should be committed with `LF` line endings.

```

Comment:
```suggestion
* Files should be committed with `LF` line endings.

```

Reply:
I think there are exceptions here (e.g. Windows binaries)

Reply:
Agreed, but those don't have any line endings, so the symbol used is irrelevant. The reason I suggested the alternative is that when I read that, first thing my mind asked is, which files should then be committed with CRLF if most are LF?

Reply:
Regardless, it's a very minor personal preference thing. Feel free to ignore it.

Reply:
```suggestion
* Text files should be committed with `LF` line endings.
```

Reply:
>Agreed, but those don't have any line endings,

Not always true, running line ending normalization on binary files can often corrupt them on windows

Reply:
>which files should then be committed with CRLF if most are LF?

Essentially just Windows compiled binaries